text/0000-selecting-default-java-version.md

```diff
@@ -10,7 +10,7 @@ Currently, there is no process how the default version of Java is picked in the
 
 ## Detailed Explanation
 
-The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. 
+The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. Setting this as the baseline, exceptions of faster (or slower) updates are acceptable in urgent cases.
```

Comment:
```suggestion
The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest released LTS version of Java is at least one year old. Exceptions are permitted for a shorter change cycle when deemed sufficiently urgent by a majority vote of Java subteam contributors and maintainers.
```

I would really recommend having the exception process spelled out more. The purpose of this RFC is to provide a stable process on which folks can depend. A predictable change cycle is a big part of that. 

If we say that the version can be changed in a shorter period of time, but does not provide any requirements then it makes the process opaque and erodes the comfort we're trying to provide for users.

This specifically says that it cannot be done by just a small number of folks. Technically two maintainers, one to commit and one to approve, could decide to change this unless we specify otherwise. Specifically saying that it requires a majority vote, makes the process harder and something most of the Java maintainers & contributors need to agree upon.